ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/HeaderBasedProtocolSchedule.java

```diff
@@ -34,4 +36,8 @@ public interface HeaderBasedProtocolSchedule {
   String listMilestones();
 
   Stream<Long> streamMilestoneBlocks();
+
+  boolean anyMatch(Predicate<ScheduledProtocolSpec> predicate);
+
+  boolean isOnForkBoundary(final BlockHeader blockHeader);
```

Comment:
The name is inconsistent with the other naming using fork instead of milestone. So suggest `isOnMilestoneBoundary`

Reply:
good spot - done